### ISSUE TITLE
Missing HTML escaping in truncated assays description

### ISSUE THREAD

**Issue by stain** [seek4science/seek#76]
About half-way down the page of https://fairdomhub.org/assays?page=A we see that a `<b>` or `<strong> `tag is left accidentally open from a truncated description, rendering every assay after in bold (Firefox 66.0.2). Similar seems to happen with `<i>` or `<em>`

Assay that causes it in that one is https://fairdomhub.org/assays/816 - it all depends on where the truncation happen.

**Comment by stuzart** [comment]
Thanks for pointing this out, I've applied a quick workaround whilst we look into a better solution

**Comment by alaninmcr** [comment]
@stuzart Was a better or permanent solution found?

**Comment by stuzart** [comment]
no, but the simple fix seems to have solved it without any adverse side effects
